knowlg-app/src/app/home/contentlist/contentlist.component.ts

```diff
@@ -10,12 +12,22 @@ import { ContentService } from '../services/content.service';
 })
 export class ContentlistComponent implements OnInit {
   contentList = [];
+  contents = [];
   mimeType: any;
   constructor(public router: Router, public contentService: ContentService, public activatedRoute: ActivatedRoute) { }
 
   ngOnInit() {
     this.mimeType = this.activatedRoute.snapshot.paramMap.get('mimeType');
     this.contentList = this.contentService.getContenstList(this.mimeType);
+    
+    this.contentService.search(this.mimeType).subscribe((data: any) => {
```

Comment:
Why do you need the above api call. The required contents based on mime type are getting using the below search call.

Reply:
If API call fails we will have sample content to show in offline for mobile app so i am including them

Reply:
It should be fallback mechanism in that case. If search is failed, then use the default.

Even search returns empty data with success, then you should not show default. It should be empty list.
Only when API fails, then use the default.